extension/docs/Refund.md

```diff
@@ -181,6 +181,12 @@ Follow these recommendations when using the idempotency key:
 - Generate idempotency keys using the version 4 (random) UUID type to prevent two API credentials under the same account from accessing each others responses.
```

Comment:
```suggestion
- Generate idempotency keys using the version 4 (random) UUID type to prevent two API credentials under the same account from accessing each other's responses.
```